test/datahike/test/gc_test.cljc

```diff
@@ -43,12 +43,12 @@
         ]
     (testing "Test initial store counts."
       (is (= 1 (count (-mark (:eavt @conn)))))
-      (is (= fresh-count (count-store @conn)))
```

Comment:
You should set `num-roots` to 3 here instead of messing with the counts directly since you now structurally share the initial indices and hence have only 3 of them with or without history.